spi/src/main/java/org/eclipse/microprofile/reactive/streams/spi/Stage.java

```diff
@@ -596,4 +597,75 @@
         CompletionStage<?> getCompletionStage();
     }
 
+    /**
+     * A stage that couples a wrapped subscriber graph to a wrapped publisher graph.
+     * <p>
+     * The resulting stage sends all the elements received to the passed in subscriber, and the emits all the elements
```

Comment:
Typo - should be "and then"  instead of "and the"

Reply:
Fixed in 5 places. Actually, it should be "and" instead of "and the".